src/schema/v2/me/myCollectionUpdateArtworkMutation.ts

```diff
@@ -136,6 +141,7 @@ export const myCollectionUpdateArtworkMutation = mutationWithClientMutationId<
         artists: artistIds,
         cost_currency_code: costCurrencyCode,
         cost_minor: costMinor,
+        artwork_artwork_location: artworkLocation,
```

Comment:
**thought:** is this intentionally called `artwork_artwork_location`

Reply:
This should be 

```suggestion
        artwork_location: artworkLocation,
```